cells/std/lib/default.nix

```diff
@@ -4,7 +4,61 @@
 }: let
   nixpkgs = inputs.nixpkgs;
 in {
-  inherit (inputs.devshell.legacyPackages) mkShell;
+  mkShell = configuration: let
+    nixagoModule = {
+      config,
+      lib,
+      ...
+    }:
+      with lib; let
+        cfg = config.devshell;
+      in {
+        options.nixago = mkOption {
+          type = types.listOf types.attrs;
+          default = [];
+          apply = x: x.passthru;
+          description = "List of `std` Nixago pebbles to load";
+        };
+
+        config.devshell = mkIf (cfg.nixago != []) {
+          startup.nixago-setup-hook = stringBefore ["motd"] inputs.nixago.lib.makeAll cfg.nixago;
```

Comment:
@jmgilman One issue here, that becomes immediatly evident is that `nixago`'s common hook doesn't cleanup after itself, now does it preserve eventual competing funtion definions of e.g. `debug`.

I'm not entirely sure what's the best way to preserve the original functions if they exist and then only temporarily load the nixago ones, `unset -f ` them at the end and restore the originals...

Reply:
for context, we talk about these: https://github.com/nix-community/nixago/blob/master/lib/hooks/common.nix

Reply:
fixed in https://github.com/nix-community/nixago/pull/46